src/chain/script/script.cpp

```diff
@@ -854,7 +854,7 @@ operation::list script::to_pay_multisig_pattern(uint8_t signatures,
 script_pattern script::pattern() const
 {
     // The first operations access must be method-based to guarantee the cache.
-    if (is_null_data_pattern(operation::list()))
+    if (is_null_data_pattern(operations()))
```

Comment:
Right.  It was an easy bug to miss because it's syntactically correct and the comment was mis-leading ;-)